src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx

```diff
@@ -63,6 +66,11 @@ export class DataSourceAggregatedView extends React.Component<
     this._isMounted = true;
     getDataSourcesWithFields(this.props.savedObjectsClient, ['id', 'title', 'auth.type'])
       .then((fetchedDataSources) => {
+        if (fetchedDataSources?.length === 0) {
+          this.setState({
+            showEmptyState: true,
```

Comment:
is it possible that we have local cluster?

Reply:
updated

Reply:
is it possible that there is data source but filtered out?